content/applications/finance/fiscal_localizations/belgium.rst

```diff
@@ -0,0 +1,169 @@
+=======
+Belgium
+=======
+
+.. _belgium/introduction:
+
+Fiscal certification: POS restaurant
+====================================
+
+In Belgium, the owner of a cooking business such as a restaurant or food truck is required by law to
+use a government-certified **Cash Register System** for their receipts. This applies if their yearly
+earnings (excluding VAT, drinks, and take-away food) exceed 25,000 euros.
+
+This government-certified system entails the use of a :ref:`certified POS system
+<belgium/certified>`, along with a device called a :ref:`Fiscal Data Module <belgium/fdm>` (or
+**black box**) and a :ref:`VAT Signing Card <belgium/vat>`.
+
+.. important::
+   Do not forget to register as *foodservice industry manager* on the `Service Public Federal
+   Finances <https://www.systemedecaisseenregistreuse.be/fr/enregistrement>`__ website.
+
+.. _belgium/certified:
+
+Certified POS system
+--------------------
+
+The Odoo POS system is certified for the latest three major versions of databases hosted on **Odoo
+Online** and **Odoo.sh**. Please refer to the following table to ensure that your POS system is
+certified.
```

Comment:
I suggest mentioning "supported versions" and linking it to the custom anchor `supported_versions`